compiler/test/formatter_outputs/chained.gr

```diff
@@ -0,0 +1,23 @@
+let g = 7, // g represents the precision desired, p is the values of p[i] to plug into Lanczos' formula
+p = 8, // another comment
+q = 32
```

Comment:
Like @marcusroberts mentioned in discord, it would be ideal if non-function bindings were indented to line-up with `let` but I don't want functions formatted that way, so I'm fine with this.

Reply:
Could we special case functions? I agree that it would look bad for functions, but I'm not a fan of how this looks not indented.

Reply:
I now indent everything that's not a lambda.